client/state/jetpack-connect/reducer.js

```diff
@@ -118,7 +118,7 @@ export function jetpackConnectAuthorize( state = {}, action ) {
 			return Object.assign( {}, state, { autoAuthorize: false } );
 		case SERIALIZE:
 		case DESERIALIZE:
-			return {};
+			return state;
```

Comment:
are we sure that we want to persist the state of the authorization? is there any reason we need to persist anything? because having in account that most of the info stored there have a quite short TTL, it can have some odd effects once it's stale